cmd/juno/juno.go

```diff
@@ -53,6 +54,7 @@ const (
 	defaultP2p                 = false
 	defaultP2pAddr             = ""
 	defaultP2pBootPeers        = ""
+	defaultMetricsPort         = 0
```

Comment:
Since we are using Prometheus to expose metrics we should by default set it to `9090`, right?

Reply:
done